ansible/playbooks/openadkit.yaml

```diff
@@ -24,6 +24,8 @@
       when: module == 'base'
     - role: autoware.dev_env.kisak_mesa
       when: module == 'base'
+    - role: autoware.dev_env.pacmod
+      when: module == 'base'
```

Comment:
The original condition was `when: module == 'planning-control' or module == 'perception-localization' or module == 'all'`. What was the difference between `base` and `all`?

Reply:
As future task, we can remove pacmod to be installed as default since it is not necessary for the users that are not using AStuff vehicles.